lua/precognition/init.lua

```diff
@@ -355,6 +355,13 @@ function M.setup(opts)
 
     ns = vim.api.nvim_create_namespace("precognition")
     au = vim.api.nvim_create_augroup("precognition", { clear = true })
+
+    if type(config.highlightColor) == "table" then
+      vim.api.nvim_set_hl(ns, "precognition", config.highlightColor)
```

Comment:
I think you're right, this should probably use the global namespace.

Also, mind capitalizing the hlgroup name? I don't think it actually matters, but pascal-case is generally used for hlgroups.

Reply:
If I use the global namespace, it doesn't seem to apply the colors provided. I can see the lightlight is defined, `:filter precog highlight` but just with a grey color instead of my defined yellow. However, if I reload the plugin via `:Lazy reload precog` (where `precog` is the name I gave to the Lazy spec for this plugin), it then uses the correct colors. I can't find anything further on the correct way to do this, after a bit of searching. Any ideas or pointers?

Reply:
Can you test the latest refactors i've done. It seems to work for me good now